apps/meteor/app/ui-message/client/popup/components/composerBoxPopup/ComposerBoxPopupCannedResponse.tsx

```diff
@@ -13,7 +13,9 @@ const ComposerPopupCannedResponse = ({ shortcut, text }: ComposerBoxPopupCannedR
 			<OptionColumn>
 				<strong>{shortcut}</strong>
 			</OptionColumn>
-			<OptionContent>{text}</OptionContent>
+			<Box withTruncatedText flexGrow={1} width={0}>
```

Comment:
flexGrow 1 + width 0 is to prevent the text from affecting the width of the flex container and still retain the ellipsis behavior.

Reply:
Guys, we shouldn't use custom componentization inside our build-in components, `<OptionContent />` should have the ellipsis behavior by default for long texts, if it's being used correctly, if not we should consider checking whats wrong and fixing it on the Fuselage side! Please let's consider this for all similar cases.

Reply:
Changed my approach. Thanks for the heads up.